### DISCUSSION TITLE
Override  Components Views

### DISCUSSION THREAD

**Discussion by AEK-BKF** [rappasoft/laravel-livewire-tables#243] [Feature Requests]
Hello,
Is there a way to override "search input" , " Bulk Action " and " PerPage"  components ? 
I may need to change html code or css.

Thanks

**Comment by bdelamatre** [comment]
Yes. You should be able to override views like any other Laravel package.

https://laravel.com/docs/8.x/packages#views

**Comment by AEK-BKF** [comment]
Thanks for reply, but what I mean, a method like filter View to override bulk action ...

**Reply by bdelamatre** [reply]
I don't follow how that's functionally different, but the only method to adjust a view template specifically I think is rowView.

Edit: Oh, right, I forgot about the filter view as well: https://github.com/rappasoft/laravel-livewire-tables/wiki/Custom-filter-view

**Reply by AEK-BKF** [reply]
Yes this is what I need for bulk actions, search and perpage components, something like this :

public function bulkView(): ?string 
{ 
      return 'path.to.my.bulk.view'; 
}

**Reply by bdelamatre** [reply]
Got it. Will you open a feature request for that specific feature with that example?

Alternatively,  @rappasoft is excellent about pull requests, if you want to submit something.
